sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -153,9 +153,9 @@ def tune(
         self,
         # TODO (andreyvelich): How to be consistent with other APIs (name) ?
         name: str,
-        objective: Callable,
+        objective: Union[Callable, str],
         parameters: Dict[str, Any],
-        base_image: str = constants.BASE_IMAGE_TENSORFLOW,
+        #base_image: str = constants.BASE_IMAGE_TENSORFLOW,
```

Comment:
I think, we should keep the `base_image`, since we use it when user set `objective` as train function.